### PR TITLE
Solana program deployment failed because blockhash was expired during…

### PR DESCRIPTION
**Pull request by godmodegalactus**
… simulation, So we ignore these types of errors as we are going to retry with new blockhash anyways

#### Problem
While deploying a big program the blockhash expires, and rpc during simulation will fail with message BlockhashNotFound. This makes the program deployment fail.

#### Summary of Changes

Ignoring the BlockhashNotFound transaction error as cli will retry to send the transaction with newer Blockhash anyways.

Fixes #

### CODE REVIEW

File: client/src/send_and_confirm_transactions_in_parallel.rs [hunk 1]
```diff
@@ -205,8 +220,7 @@ async fn send_transaction_with_rpc_fallback(
                     // fall through on io error, we will retry the transaction
```
Comment (joncinque) [inline client/src/send_and_confirm_transactions_in_parallel.rs:220]: nit: I can't suggest in the line above this, but you could simplify this implementation by just matching the cases in the line above, and not need `add_transaction_error_in_context`, ie:

```
ErrorKind::Io(_) | ErrorKind::Reqwest(_) |
ErrorKind::TransactionError(TransactionError::BlockhashNotFound) | 
ErrorKind::RpcError(RpcError::RpcResponseError { data: RpcResponseErrorData::SendTransactionPreflightFailure(RpcSimulateTransactionResult { err: Some(TransactionError::BlockhashNotFound, .. } ) } )
```

What do you think?

**Review by joncinque — COMMENTED** [review]
Thanks for submitting the fix! Just a little nit to avoid multiple matches to make the fix clearer (I hope). Let me know what you think!

File: client/src/send_and_confirm_transactions_in_parallel.rs (hunk 1 above)
Comment (godmodegalactus) [reply client/src/send_and_confirm_transactions_in_parallel.rs:220]: I think you are right
let me do the changes and commit it back.

**Review by joncinque — APPROVED** [review]
Looks great, thanks!

**Pull request merged**